projects/showcase/src/app/doc/doc.module.ts

```diff
@@ -4,11 +4,11 @@ import { FormsModule } from '@angular/forms';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatSelectModule } from '@angular/material/select';
 import { RouterModule, Routes } from '@angular/router';
-import { NgxMdModule } from 'ngx-md';
-import 'prismjs/components/prism-typescript';
-import 'prismjs/prism';
+import { MarkdownModule } from 'ngx-markdown';
 import { LayoutModule } from '../shared/layout/layout.module';
 import { DocComponent } from './doc.component';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import 'prismjs/components/prism-typescript.min.js';
```

Comment:
Shouldn't this line be inside the `angular.json` file in the scripts?

Reply:
I just applied the same change from upstream. Actually i quite don't understand the need for this `prismjs-typescript` import.
Moreover it seems that the showcase/doc app works equally well without it.
But i'd leave it like that, just for the sake of keeping minimum changes with upstream.

Reply:
Fair enough, worth opening an issue with the upstream repo to understand what this is for.
Thanks again!!!